plugins/react.js

```diff
@@ -44,6 +44,10 @@ module.exports = {
       1,
       { namedComponents: 'arrow-function' }
     ],
+    // ensure destructuring and symmetric naming of useState hook value and setter variables
+    'react/hook-use-state': 0,
```

Comment:
I would encourage this to be a `1` warning. All `useState` calls should, ideally, conform to 1) a destructured pair and 2) consistent `[thing, setThing]` naming. This follows the general direction of Garden's config.